src/docview.ts

```diff
@@ -69,7 +73,27 @@ export class DocViewProvider implements Disposable {
                 link.attribs.href = mkCommandUri('lean.openDocView', href);
             }
         }
-        this.getWebview().webview.html = $.html();
+        $(`<nav style="box-sizing: border-box; position : sticky; top : 0px; padding : 4px; background : white">
```

Comment:
Maybe we could have back/forward as buttons in the title bar instead?  You just need to add the commands to the `editor/title` menu in `package.json`: https://github.com/leanprover/vscode-lean/blob/master/package.json#L358

Then `"when"` clauses are a bit broken since we've switched to webview (from the previous previewHTML command).  For example here they should only show the button on infoview tabs: https://github.com/leanprover/vscode-lean/blob/master/package.json#L375  This does not work now (`resourceScheme` doesn't have the right value).

We'd need to set a context key like the markdown extension now, that indicates whether the docview is active or not: https://github.com/Microsoft/vscode/blob/1b61a70042a75e5f7273227b0974ffef51894adc/extensions/markdown-language-features/package.json#L104-L108 https://github.com/Microsoft/vscode/blob/master/extensions/markdown-language-features/src/features/previewManager.ts#L152-L154

Unfortunately, this option has the disadvantage that you need to click on the docview tab first, otherwise the buttons are not shown.